src/healthkit.cs

```diff
@@ -564,7 +564,7 @@ interface HKCorrelationType {
 
 	delegate void HKHealthStoreGetRequestStatusForAuthorizationToShareHandler (HKAuthorizationRequestStatus requestStatus, NSError error);
 	delegate void HKHealthStoreRecoverActiveWorkoutSessionHandler (HKWorkoutSession session, NSError error);
-	delegate void HKHealthStoreCompletionHandler (bool success, [NullAllowed] NSError error);
+	delegate void HKHealthStoreCompletionHandler (bool success, NSError error);
```

Comment:
Weird... what was the compiler issue?

Reply:
Thanks for bringing this up @SotoiGhost !
Here is a gist of the errors / warnings that were not present when not including the [NullAllowed]
https://gist.github.com/tj-devel709/756ac73e32f3ff8ae234d8ba7ecdd7cc